src/AlgoliaSearch.js

```diff
@@ -103,6 +103,7 @@ function AlgoliaSearch(applicationID, apiKey, opts) {
 
   this._ua = opts._ua;
   this._useCache = opts._useCache === undefined || opts._cache ? true : opts._useCache;
+  this._useFallback = opts.useFallback === undefined ? true : opts.useFallback;
```

Comment:
I try to avoid "smart" notations where you have to double think what's it's doing to the provided value